source/features/repo-header-info.tsx

```diff
@@ -49,10 +49,12 @@ async function add(repoLink: HTMLAnchorElement): Promise<void> {
 			<a
 				href={buildRepoURL('stargazers')}
 				title={`Repository starred by ${stargazerCount.toLocaleString('us')} people`}
-				className="d-flex flex-items-center flex-justify-center mr-1 gap-1 color-fg-muted"
+				className="d-inline-block"
 			>
-				<StarIcon className="ml-1" width={12} height={12}/>
-				<span className="f6">{abbreviateNumber(stargazerCount)}</span>
+				<div className="d-inline-flex flex-items-center flex-justify-center mr-1 gap-1 v-align-text-top color-fg-muted">
```

Comment:
Can't you avoid the wrapper `<div>` by applying exactly these classes to `<a>`?

Reply:
Thank you!
I thought It was not changed when without wrapping because of my old eyes issue.